demo/src/main/resources/application.properties

```diff
@@ -2,7 +2,18 @@ spring.datasource.url=jdbc:mysql://localhost:3306/booking?serverTimeZone=UTC
 spring.datasource.username=root
 spring.datasource.password=Password
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-
 spring.jpa.hibernate.ddl-auto=create-drop
+
 spring.jpa.show-sql=true
 server.port=8080
+
+jwt.expiration=300000
+jwt.secret=gerhjthrtjhgf8fddg44gdgd9gbgfdhythjliuuvsgd
+
+bot_name=Booking
+bot_token=7248708324:AAHkcibCFZWcmrDLx0biR22plp_qJjb3ONE
+
+stripe.api.key=sk_test_51PjiL92NMR41Gk6RlzzRMZ13Im406QMpxbZUeHB9BOu\
+  3gWVlhvaxRBxLbSX9sxGv3WQlRTi7uE2VT082dRzPqblQ005ehW1egH
```

Comment:
i would extract these to .env too
you will need to add 
`spring.config.import=optional:file:.env[.properties]`